packages/sui-lint/.prettierrc.js

```diff
@@ -1,13 +1,22 @@
 module.exports = {
-  printWidth: 80,
-  tabWidth: 2,
-  singleQuote: true,
-  trailingComma: 'none',
+  /* Include parentheses around a sole arrow function parameter. */
+  arrowParens: 'avoid',
+  /* Print spaces between brackets in object literals. */
   bracketSpacing: false,
-  semi: false,
-  useTabs: false,
-  parser: 'babel',
+  /* Put the > of a multi-line JSX element at the end of the last line instead of being alone on the next line */
   jsxBracketSameLine: false,
-  arrowParens: 'avoid',
-  optionalChaining: true
+  /* Support optional chaining new ES feature -> ? */
+  optionalChaining: true,
+  /* Specify the line length that the printer will wrap on. */
+  printWidth: 80,
+  /* Print semicolons at the ends of statements. */
+  semi: false,
+  /* Use single quotes instead of double quotes. */
+  singleQuote: true,
+  /* Specify the number of spaces per indentation-level. */
+  tabWidth: 2,
+  /* Print trailing commas wherever possible when multi-line.  */
+  trailingComma: 'es5',
```

Comment:
👀

Reply:
Well, the code will work no matter this. In fact, the linter shouldn't give you an error with or without the trailing comma. But every-time prettier format your code, it will be formatted different (with trailing comma). It's a styling change and everything should keep work as expected.

Reply:
I could remove the change to keep the old behavior. :) It's true some linter error could show up but prettier should fix automatically.